web-test-runner-firefox.config.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-env node */
+require('dotenv').config();
```

Comment:
We only need Sauce env variables for Lumo and Material builds.
Do you think it would be good to still have this here for consistency?

Reply:
Sorry, I should have commented on that. This was just to prepare for the case when we need to add additional variables in the future that might then be used in the other scripts. What I wanted to avoid is that someone follows the instructions in the README and then it still doesn't work because the variables are not picked up by the other scripts.

Reply:
Makes sense, thanks.